### PR TITLE
Remove Nightly Connect from the selector

### PR DESCRIPTION
**Pull request by Pawel-Szydlo**
# Description

Please include a summary of the wallet and how users can install it. Be sure to take a look at our guide on [Custom Wallets](https://github.com/near/wallet-selector/blob/main/packages/core/docs/guides/custom-wallets.md) to understand core concepts when integrating with Wallet Selector.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +66,7 @@ yarn add \
   @near-wallet-selector/near-snap \
   @near-wallet-selector/ledger \
   @near-wallet-selector/wallet-connect \
-  @near-wallet-selector/nightly-connect \
+  @near-wallet-selector/default-wallets \
```
Comment (kujtimprenkuSQA) [inline README.md:69]: This line should be removed, `default-wallets` package has been deprecated a few months ago:
https://www.npmjs.com/package/@near-wallet-selector/default-wallets

File: README.md [hunk 2]
```diff
@@ -90,7 +89,7 @@ npm install \
...
```
Comment (kujtimprenkuSQA) [inline README.md:92]: This line should be removed, `default-wallets` package has been deprecated a few months ago:
https://www.npmjs.com/package/@near-wallet-selector/default-wallets

**Review by kujtimprenkuSQA — COMMENTED** [review]
Hi, thank you for submitting this PR.

Why is the `nightly-connect` package being removed?
Has there been any announcement from Nightly about this?

The community should be aware that this option will no longer be maintained from Nightly if that's the case.

If this package needs to be removed before that we would need to publish another version of this package and update the readme file that would show up here on NPM:

https://www.npmjs.com/package/@near-wallet-selector/nightly-connect

The updated readme should have information that describes why Nightly Connect is deprecated, and no longer maintained.

> Note: We will not remove the package from NPM even if this PR gets merged, we will just deprecate all versions of this package.

**Comment by kujtimprenkuSQA** [comment]
> If this package needs to be removed before that we would need to publish another version of this package and update the readme file that would show up here on NPM:
> 
> https://www.npmjs.com/package/@near-wallet-selector/nightly-connect
> 
> The updated readme should have information that describes why Nightly Connect is deprecated, and no longer maintained.

This suggestion can be ignored if this package is to be completely removed from the repo, the message on NPM should be enough when we mark it as deprecated.

Can you please address other comments?

**Comment by Pawel-Szydlo** [comment]
> Hi, thank you for submitting this PR.
> Why is the `nightly-connect` package being removed? Has there been any announcement from Nightly about this?
> The community should be aware that this option will no longer be maintained from Nightly if that's the case.
> If this package needs to be removed before that we would need to publish another version of this package and update the readme file that would show up here on NPM:
> https://www.npmjs.com/package/@near-wallet-selector/nightly-connect
> The updated readme should have information that describes why Nightly Connect is deprecated, and no longer maintained.
> > Note: We will not remove the package from NPM even if this PR gets merged, we will just deprecate all versions of this package.

The marketing team is currently preparing a post about nightlyConnect-v1. The packages have not been updated for a while and we have significantly improved the nightlyConnect performance. I think that within a week users will be informed about the lack of support for nightlyConnect-v1

We have created a new nightlyConnect-v2 which is faster and easy to integrate, more advantages will be presented by the nightly marketing team.

**Comment by Pawel-Szydlo** [comment]
@kujtimprenkuSQA So I have to undo the deletion of the nightly-connect folder and in the readme add the appropriate note? understood correctly?

**Comment by kujtimprenkuSQA** [comment]
> @kujtimprenkuSQA So I have to undo the deletion of the nightly-connect folder and in the readme add the appropriate note? understood correctly?

If `nightly-connect` needs to be removed then the PR can stay as it is, but the individual comments need to be addressed:
https://github.com/near/wallet-selector/pull/984#discussion_r1374368695
https://github.com/near/wallet-selector/pull/984#discussion_r1374368522

**Comment by kujtimprenkuSQA** [comment]
> > Hi, thank you for submitting this PR.
> > Why is the `nightly-connect` package being removed? Has there been any announcement from Nightly about this?
> > The community should be aware that this option will no longer be maintained from Nightly if that's the case.
> > If this package needs to be removed before that we would need to publish another version of this package and update the readme file that would show up here on NPM:
> > https://www.npmjs.com/package/@near-wallet-selector/nightly-connect
> > The updated readme should have information that describes why Nightly Connect is deprecated, and no longer maintained.
> > > Note: We will not remove the package from NPM even if this PR gets merged, we will just deprecate all versions of this package.
> 
> The marketing team is currently preparing a post about nightlyConnect-v1. The packages have not been updated for a while and we have significantly improved the nightlyConnect performance. I think that within a week users will be informed about the lack of support for nightlyConnect-v1
> 
> We have created a new nightlyConnect-v2 which is faster and easy to integrate, more advantages will be presented by the nightly marketing team.

Just to clarify here, does this mean if the package `nightly-connect` is deleted Wallet Selector will no longer support any version of Nightly Connect?

**Comment by kujtimprenkuSQA** [comment]
Can you change the base branch from `main` to `dev` as we do not merge PRs to main unless it's for a release?

**Comment by Pawel-Szydlo** [comment]
> Just to clarify here, does this mean if the package `nightly-connect` is deleted Wallet Selector will no longer support any version of Nightly Connect?

We anticipate support for the Near network in nightlyConenct-v2, however, this may still take some time.

But our Business developer will contact you about this - next week.

**Comment by Pawel-Szydlo** [comment]
Improved @kujtimprenkuSQA

**Review by kujtimprenkuSQA — APPROVED** [review]

**Pull request merged**